presto-hive/src/main/java/com/facebook/presto/hive/HiveTableProperties.java

```diff
@@ -77,7 +79,8 @@ public HiveTableProperties(TypeManager typeManager, HiveClientConfig config)
                                 .map(name -> ((String) name).toLowerCase(ENGLISH))
                                 .collect(Collectors.toList())),
                         value -> value),
-                integerSessionProperty(BUCKET_COUNT_PROPERTY, "Number of buckets", 0, false));
+                integerSessionProperty(BUCKET_COUNT_PROPERTY, "Number of buckets", 0, false),
+                stringSessionProperty(LOCATION, "physical file system location", config.getLocation(), false));
```

Comment:
Use `null` for the default. If it's not specified, we should use the location from the Hive database object.